backend/app/Actions/GeoLocation/GetGeoLocationItemsAction.php

```diff
@@ -79,7 +79,7 @@ public function execute(GetGeoLocationItemsRequest $request): GetGeoLocationItem
                 $item['all_visitors_count'],
                 $item['new_visitors_count'] ?? 0,
                 $item['all_sessions_count'],
-                $item['bounced_visitors_count'] ?? 0 / $item['all_visitors_count'] ?? 1,
+                isset($item['bounced_visitors_count'])? $item['all_visitors_count'] / $item['bounced_visitors_count'] : 0,
```

Comment:
the relation should be vice versa
$item['bounced_visitors_count'] / $item['all_visitors_count']

and put it to variable, because the line is too long